tests/Aspire.Hosting.Python.Tests/AddPythonProjectTests.cs

```diff
@@ -51,9 +54,9 @@ public async Task AddInstrumentedPythonProjectProducesDockerfileResourceInManife
 
         var manifestPath = Path.Combine(projectDirectory, "aspire-manifest.json");
 
-        using var builder = TestDistributedApplicationBuilder.Create(options =>
+        using var builder = CreateTestDistributedApplicationBuilder(options =>
         {
-            options.ProjectDirectory = Path.GetFullPath(projectDirectory);
```

Comment:
Note that this property is internal, and we don't want to use InternalsVisibleTo for extension tests. So I used reflection instead